src/actions/ast.js

```diff
@@ -39,7 +39,7 @@ export function setSourceMetaData(sourceId: SourceId) {
     }
 
     const framework = await getFramework(source.id);
-    dispatch(updateTab(source.url, framework));
+    dispatch(updateTab(source, framework));
```

Comment:
i think we can move `const framework = await getFramework(source.id);` into `updateTab` so that whenever we call update it does this regardless of where we call it from

Reply:
So I looked at this and I actually like where it is now.  This meta gets set once and it's when we know what that framework info is.  We don't want to be grabbing framework info every time that `updateTab` is called.